src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -215,61 +245,78 @@ dependencies {
 ```
 
 {{site.alert.important}}
-If you're located in China, use a mirror site such as
-`https://[a mirror site]/download.flutter.io` rather than the
-`storage.googleapis.com` domain directly. See our
-[Using Flutter in China](/community/china) page for information on mirrors.
+  If you're located in China, use a mirror site such as
+  `https://[a mirror site]/download.flutter.io` rather than the
+  `storage.googleapis.com` domain directly. See our
+  [Using Flutter in China][] page for information on mirrors.
 {{site.alert.end}}
 
 {{site.alert.tip}}
-You can also build an AAR for your Flutter module in Android Studio using
-the `Build > Flutter > Build AAR` menu.
+  You can also build an AAR for your Flutter module in Android Studio using
+  the `Build > Flutter > Build AAR` menu.
 
 {% include app-figure.md image="development/add-to-app/android/project-setup/ide-build-aar.png" %}
 {{site.alert.end}}
 
-Your app now includes the Flutter module as a dependency. You can follow the
-next steps in the [API usage documentations](/docs/development/add-to-app/android/add-flutter-screen).
+  Your app now includes the Flutter module as a dependency.
+  You can follow the next steps in the
+  [API usage documentation][].
```

Comment:
This links to "Adding a Flutter screen to an Android app." Is "API usage documentation" the best description we can manage? (I think I expected it'd go to generated API docs.)